### DISCUSSION TITLE
A problem with Wind Down

### DISCUSSION THREAD

**Discussion by DeathManeYT-ModCode** [ppy/osu#21791] [Q&A]
### Type

Game behaviour

### Bug description

I was using WD and I noticed that notes that weren't there randomly appeared, said notes are normally there without WD.
I thought it was just an off-hand thing but it happened on 2 maps consecutively. I checked the replay and the notes were doing the same thing in the replay. I had to upload the replay to mediafire because of Github not accepting osr files.

### Screenshots or videos

https://www.mediafire.com/file/1td1vwjz0abkebg/DeathMane_playing_Ice_-_Entrance_%2528leqek%2529_%255B4K_Insane%255D.osr/file

### Version

2022.1208.0-lazer

### Logs

[performance-update.log](https://github.com/ppy/osu/files/10300824/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/10300825/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10300826/updater.log)
[database.log](https://github.com/ppy/osu/files/10300827/database.log)
[input.log](https://github.com/ppy/osu/files/10300828/input.log)
[network.log](https://github.com/ppy/osu/files/10300829/network.log)
[performance.log](https://github.com/ppy/osu/files/10300830/performance.log)
[performance-input.log](https://github.com/ppy/osu/files/10300831/performance-input.log)

**Comment by frenzibyte** [comment]
We can't deal with replay files here. Attach a video record of the issue at hand. Also I don't understand the issue here, you mention there are notes that "randomly appear with WD", but then contradict that by saying "said notes normally appear without WD"..?

**Comment by DeathManeYT-ModCode** [comment]
osu! 2022-12-25 19-54-50.mp4
<https://drive.google.com/file/d/1sE0WYOipBwK_w8bXWrBRGKNx52fRe6h8/view?usp=drive_web>
I meant that the notes are there without wind down, but they're invisible
until last second with WD. The video is attached here. Took me a bit to
figure out how to record it with Gamebar.

**Reply by frenzibyte** [reply]
Video is private, can't access it.

**Comment by DeathManeYT-ModCode** [comment]
I think I fixed it.
https://drive.google.com/file/d/1sE0WYOipBwK_w8bXWrBRGKNx52fRe6h8/view?usp=share_link
